### PR TITLE
Closes #1907 rename query data set variables

### LINKED ISSUE #1907: Feature Request:  Update names on Query Dataset so they are <= 8 characters
### Feature Idea

Current Query data set has names >8 characters so will cause an issue when submitting a filing. Update those names and update the rest to be consistent.

### Relevant Input

See https://pharmaverse.github.io/admiral/cran-release/articles/queries_dataset.html

### Relevant Output

Change variables as follows:
VAR_PREFIX to PREFIX
QUERY_NAME to GRPNAME
TERM_LEVEL to SRCVAR
TERM_NAME to TERMNAME
TERM_ID to TERMID
QUERY_ID to GRPID
QUERY_SCOPE to SCOPE
QUERY_SCOPE_NUM to SCOPEN
VERSION to VERSION (no change)

### Reproducible Example/Pseudo Code

_No response_

### PR DESCRIPTION
**Pull request by millerg23**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiraldev/devel/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/devel/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/cran-release/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

File: docs/pkgdown.yml [hunk 1]
```diff
@@ -19,8 +19,8 @@ articles:
   queries_dataset: queries_dataset.html
   questionnaires: questionnaires.html
   visits_periods: visits_periods.html
-last_built: 2023-05-11T13:11Z
+last_built: 2023-05-22T15:01Z
 urls:
-  reference: https://pharmaverse.github.io/admiral/cran-release/reference
-  article: https://pharmaverse.github.io/admiral/cran-release/articles
+  reference: https://pharmaverse.github.io/admiral/reference
```
Comment (zdz2101) [inline docs/pkgdown.yml:24]: Don't think we want this changed

File: R/globals.R [hunk 2]
```diff
@@ -141,8 +141,8 @@ globalVariables(c(
   "SI_UNIT_UPPER",
   "VAR_CHECK",
   "TERM",
-  "TERM_LEVEL",
-  "TERM_NAME",
+  "SRCVAR",
+  "TERMNAME",
   "TERM_NAME_ID",
```
Comment (zdz2101) [inline R/globals.R:146]: Looks like "TERM_ID"/"TERMID" was never needed?

**Review by zdz2101 — COMMENTED** [review]
Really small stuff, but looks good!

File: R/globals.R (hunk 2 above)
Comment (millerg23) [reply R/globals.R:146]: Not sure what you mean, should i update this file?

**Comment by millerg23** [comment]
Thanks for reviewing this @zdz2101. Could you also review https://github.com/pharmaverse/admiral.test/pull/137 as its related to this update.

**Review by zdz2101 — APPROVED** [review]

**Review by bundfussr — CHANGES_REQUESTED** [review]

File: NEWS.md [hunk 3]
```diff
@@ -50,6 +50,18 @@ This affects `derive_var_dthcaus()`. (#1727)
...
+
+  - `VAR_PREFIX` to `PREFIX`
+  - `QUERY_NAME` TO `GRPNAME`
+  - `QUERY_ID` TO `GRPID`
+  - `QUERY_SCOPE` TO `SCOPE`
+  - `QUERY_SCOPE_NUM` TO `SCOPEN`
+  - `TERM_LEVEL` TO `SRCVAR`
+  - `TERM_NAME` TO `TERMNAME`
+  - `TERM_ID` TO `TERMID`
+
```
[the remark this answers is not in the document]
Comment (millerg23) [reply NEWS.md]: Fixed, and updated as suggested. PLease review @bundfussr

File: NEWS.md [hunk 4]
```diff
@@ -52,8 +52,23 @@ This affects `derive_var_dthcaus()`. (#1727)
...
+  - `VAR_PREFIX` to `PREFIX`
+  - `QUERY_NAME` to `GRPNAME`
+  - `QUERY_ID` to `GRPID`
+  - `QUERY_SCOPE` to `SCOPE`
+  - `QUERY_SCOPE_NUM` to `SCOPEN`
+  - `TERM_LEVEL` to `SRCVAR`
+  - `TERM_NAME` to `TERMNAME`
+  - `TERM_ID` to `TERMID`
+  
+Users need to adjust their `get_terms` function accordingly.
```
Comment (bundfussr) [inline NEWS.md:70]:
```suggestion
    query data set as follows:

    - `VAR_PREFIX` to `PREFIX`
    - `QUERY_NAME` to `GRPNAME`
    - `QUERY_ID` to `GRPID`
    - `QUERY_SCOPE` to `SCOPE`
    - `QUERY_SCOPE_NUM` to `SCOPEN`
    - `TERM_LEVEL` to `SRCVAR`
    - `TERM_NAME` to `TERMNAME`
    - `TERM_ID` to `TERMID`
  
    Users need to adjust their `get_terms()` function accordingly.
```
Update indentation such that the last sentence is rendered as part of the item.

**Review by bundfussr — CHANGES_REQUESTED** [review]

**Review by bundfussr — APPROVED** [review]

**Pull request merged**